### PR TITLE
ISSUE_TEMPLATE: please do not report outdated docs

### PR DESCRIPTION
**Pull request by simon04**

### CODE REVIEW

**Review by ojeytonwilliams — COMMENTED** [review]
The text looks fine, but can we move it to `.github/ISSUE_TEMPLATE/default.md`? It doesn't matter a great deal, but GitHub complains when you go here: https://github.com/freeCodeCamp/devdocs/blob/master/.github/ISSUE_TEMPLATE.md

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 <!--
-  Please read the contributing guidelines before opening an issue:
+  ⚠ Please do not report outdated docs, we have an automated versions report:
```
Comment (ojeytonwilliams) [inline .github/ISSUE_TEMPLATE.md:2]:
```suggestion
  ⚠ Please check the automated versions report before reporting outdated docs:
```
Since a numpy update was missed (https://github.com/freeCodeCamp/devdocs/issues/1352) how about this wording?

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (simon04) [reply .github/ISSUE_TEMPLATE.md]: Yes, your wording is better. :-)

**Comment by CzBiX** [comment]
> If the latest documentation versions report wrongly shows a documentation to be up-to-date, please open an issue or a PR to fix it.

I read this from [CONTRIBUTING.md](https://github.com/freeCodeCamp/devdocs/blob/master/.github/CONTRIBUTING.md), so I summitted an issue for outdated numpy. Hope I didn’t do anything wrong.

**Comment by ojeytonwilliams** [comment]
@CzBiX don't worry, you did the right thing.  If it's not in the report, then creating an issue (or commenting on the report) definitely helps.

**Review by ojeytonwilliams — APPROVED** [review]
